tailwind.config.cjs

```diff
@@ -14,7 +14,15 @@ module.exports = {
         silver: 'hsl(210, 6%, 72%)',
         gold: 'hsl(48, 100%, 50%)',
       },
-    },
+      typography: {
+        "code::before": {
+          content: '""',
+        },
+        "code::after": {
+          content: '""',
+        },
```

Comment:
Ah, if I’m reading some of the blog posts about this correctly, we may need to wrap this in `DEFAULT` like this:

```suggestion
        DEFAULT: {
          "code::before": {
            content: '""',
          },
          "code::after": {
            content: '""',
          },
        },
```